---
bool success
ArduinoInfo arduino
OPCUAInfo opcua